docs/sql/oracle/create-schema.sql

```diff
@@ -0,0 +1,28 @@
+/*
+ * PowerAuth Cloud
+ * Copyright (C) 2020 Wultra s.r.o.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+CREATE TABLE ES_OPERATION_TEMPLATE (
+    ID NUMBER(19) NOT NULL PRIMARY KEY,
+    PLACEHOLDER VARCHAR2(255) NOT NULL,
```

Comment:
This should be VARCHAR2(255 CHAR).

Reply:
To explain a bit better, `VARCHAR2(255)` is a `VARCHAR` with 255 bytes, which for UTF-8 is much less characters than 255. `VARCHAR2(255 CHAR)` is a `VARCHAR` with 255 characters, which is what we probably want.